zeppelin-web/src/index.html

```diff
@@ -191,6 +191,7 @@
     <script src="bower_components/ace-builds/src-noconflict/keybinding-emacs.js"></script>
     <script src="bower_components/ace-builds/src-noconflict/ext-language_tools.js"></script>
     <script src="bower_components/ace-builds/src-noconflict/theme-chrome.js"></script>
+    <script src="bower_components/ace-builds/src-noconflict/mode-javascript.js"></script>
```

Comment:
BTW, I don't know well but, is this change related to this issue directly?

Reply:
@jongyoul Mmm I build it locally to test the PR with a real Neo4j instance, IDK why it's here. Should I remove it?

Reply:
I think it would be better to minimize changes and even if it's not related to this PR directly, I think we'd better remove it.